pom.xml

```
@@ -107,12 +108,28 @@
             <artifactId>bcprov-jdk18on</artifactId>
             <version>1.72</version>
         </dependency>
+        <dependency>
+            <groupId>org.bouncycastle</groupId>
+            <artifactId>bcpkix-jdk18on</artifactId>
+            <version>1.72</version>
```

I suggest to introduce property `bc.version`
```suggestion
            <version>${bc.version}</version>
        </dependency>
        <dependency>
            <groupId>org.bouncycastle</groupId>
            <artifactId>bcpkix-jdk18on</artifactId>
            <version>${bc.version}</version>
```